### DISCUSSION TITLE
Pagination not showing

### DISCUSSION THREAD

**Discussion by Danendra10** [rappasoft/laravel-livewire-tables#1291] [Q&A]
Here is my code
<img width="674" alt="Screenshot 2023-08-04 at 17 24 35" src="https://github.com/rappasoft/laravel-livewire-tables/assets/72090618/80da5c0a-9e3d-431f-a872-52a67bdd815a">

But it doesn't show any page in it, can you please help me
<img width="1249" alt="Screenshot 2023-08-04 at 17 24 53" src="https://github.com/rappasoft/laravel-livewire-tables/assets/72090618/c74bb36b-60f0-49df-9a3e-6e41f2fc6d7c">

**Comment by lrljoe** [comment]
Okay, firstly, are you using bootstrap or tailwind on your site?

Next, what are all of those variables for please?  There are set methods for pagination in the docs that explain what you can do, if you override internal variables, then you may end up with issues due to the way that the package sets up the various elements!

I'm assuming there's a reason you're using simple pagination rather than standard?

If you can share your full configure() section, then that'd be helpful, or feel free to jump in the Discord and ping me, and I'll take a look in there.

**Reply by Danendra10** [reply]
1. I use tailwind
2. Here is the fill configure code
```php
public function configure(): void
    {
        $this->setPrimaryKey('id')
            ->setPageName('shipping-lines-export')
            ->setPaginationStatus(true)
            ->setPaginationMethod('simple')
            ->setAdditionalSelects(['shipping_lines.id as id'])
            ->setConfigurableAreas([
                'toolbar-left-start' => ['includes.areas.add-shipping-line'],
            ])
            ->setThAttributes(function ($row) {
                return ['class' => 'bg-white dark:bg-gray-700 text-black dark:text-white'];
            })
            ->setTdAttributes(function ($row) {
                return ['class' => 'px-4 py-2 text-black dark:text-white'];
            })
            ->setTrAttributes(function ($row) {
                return ['class' => 'bg-white dark:bg-gray-700 text-black dark:text-white'];
            })
            ->setTableAttributes([
                'class' => 'bg-white dark:bg-gray-700 text-black dark:text-white table-responsive'
            ]);
    }
```
3. Here is the code outside the function
```php
    protected $model = ShippingLine::class;

    public bool $perPageAll = true;
    public bool $paginationEnabled = true;
    public bool $showPerPage = true;
    public bool $showPagination = true;
```

and may i know the link of your discord?

**Reply by Danendra10** [reply]
It's all of sudden working idk why....
